src/app/components/trip-information/trip-information.component.ts

```diff
@@ -133,16 +136,17 @@ export class TripInformationComponent {
 
   assignTripData(){
 
-    this.tripNgModel.departure_date                 = this.initialForm.outbound_date;
-    this.tripNgModel.destination_city               = this.initialForm.q;
-    this.tripNgModel.return_date                    = this.initialForm.return_date;
+    this.tripNgModel.departureDate                  = this.initialForm.outbound_date;
+    this.tripNgModel.destination_city               = this.initialForm.q;  
+    this.tripNgModel.returnDate                     = this.initialForm.return_date;
     this.tripNgModel.budget                         = this.tripBudget.total;
-    // this.tripNgModel.currency                       = this.userCurrency[0]; //Falta implementar el servicio de currency
+    this.tripNgModel.currency                       = Number(this.userInfo.currencyId ?? 0);; //Falta implementar el servicio de currency
```

Comment:
You can move out the nullish default operator. It seems there is a repeated semicolon at the end.